src/javascripts/services/apiService.js

```diff
@@ -74,6 +74,13 @@ class ApiService{
         throw new Error("Failed to add transaction.");
       }
     } 
+
+    async deleteTransactionById (id) {
```

Comment:
Not have space between function name and param

Reply:
I fixed @phuocduong-agilityio . Please help me check. Thanks!